### PR TITLE
module: add spacebar service

### PR DESCRIPTION
**Pull request by cmacrae**
Another service for your consideration :) This is to acompany this PR in NixOS/nixpkgs#86556  
Once that package is generally available in the channels, the tests included in these changes should pass

### CODE REVIEW

**Comment by cmacrae** [comment]
I'll take the time tomorrow to clean up the code here like we went over in #196

**Comment by cmacrae** [comment]
Trying to determine why these tests are failing. They pass fine on my machine with `nix-build release.nix -A tests.services-spacebar`

**Comment by LnL7** [comment]
Can you rebase?

**Comment by cmacrae** [comment]
How's that? Better? :)

File: modules/services/spacebar/default.nix [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      example = literalExample ''
+        echo "spacebar config loaded..."
+      '';
+      description = ''
+        Extra arbitrary configuration to append to the configuration file.
+      '';
+    };
+  };
+
+  config = mkMerge [
```
Comment (LnL7) [inline modules/services/spacebar/default.nix:60]: no need for mkMerge here

Comment (cmacrae) [reply modules/services/spacebar/default.nix]: Good catch 👍 I've addressed it in the latest revision

**Pull request merged**